### ISSUE TITLE
A11y hints with axe-core

### ISSUE THREAD

**Issue by Gonzo17** [nuxt/hints#148]
### Describe the feature

Hi!

I really like the module so far. The first thing that came to my mind that might be missing is more information about potential A11y issues. As far as I can tell, `axe-core` is the way to go and is used in a lot of similar tools.

So how about integrating `axe-core` and show A11y issues, kind of similar like the hydration issues with highlighting the DOM element?

I'm also willing to help if you think that's fitting here. I know there is the [A11y roadmap](https://github.com/nuxt/nuxt/issues/23255), so maybe this is already in work, but I'm not sure about the state there.

### Additional information

- [x] Would you be willing to help implement this feature?
- [ ] Could this feature be implemented as a module?

### Final checks

- [x] Read the [contribution guide](https://nuxt.com/docs/4.x/community/contribution).
- [x] Check existing [discussions](https://github.com/nuxt/hints/discussions) and [issues](https://github.com/nuxt/hints/issues).

**Comment by huang-julien** [comment]
Hello ! 👋

Thank you very much for your positive review 😄 !
About a11y, we'll have a surprise for you next week (hum or in two weeks) ! ( Not in this module though)

**Comment by Gonzo17** [comment]
Hey @huang-julien, thanks for your reply, I'll keep my eyes open :)

**Comment by doppelmutzi** [comment]
I'm also curious. What was the surprise?

**Comment by joaopedrodcf** [comment]
I guess the surprise was released today with nuxt/a11y and the issue can be closed hehe

https://github.com/nuxt/a11y
